staging/src/k8s.io/kubectl/pkg/describe/versioned/describe.go

```diff
@@ -1551,6 +1551,9 @@ func describePersistentVolumeClaim(pvc *corev1.PersistentVolumeClaim, events *co
 			w.Write(LEVEL_0, "DataSource:\n")
 			w.Write(LEVEL_1, "Name:\t%v\n", pvc.Spec.DataSource.Name)
 			w.Write(LEVEL_1, "Kind:\t%v\n", pvc.Spec.DataSource.Kind)
+			if pvc.Spec.DataSource.APIGroup != nil {
```

Comment:
Should the order be large->small, or small->large? ie

* apigroup, kind, name
* name, kind, apigroup

It looks like other places do the first, so we should try to be consistent.

Reply:
I'd expect group, kind, name

Reply:
> I'd expect group, kind, name

I also expect this. wdyt? @msau42

Reply:
sgtm

Reply:
done